pkg/datarepair/checker/checker.go

```diff
@@ -187,50 +171,29 @@ func (checker *Checker) IdentifyInjuredSegments(ctx context.Context) (err error)
 	return nil
 }
 
-// Find invalidNodes by checking the audit results that are place in overlay
-func (checker *Checker) invalidNodes(ctx context.Context, nodeIDs storj.NodeIDList) (invalidNodes []int, err error) {
-	// filter if nodeIDs have invalid pieces from auditing results
+func (checker *Checker) getMissingPieces(ctx context.Context, pieces []*pb.RemotePiece) (missingPieces []int32, err error) {
+	var nodeIDs storj.NodeIDList
+	for _, p := range pieces {
+		nodeIDs = append(nodeIDs, p.NodeId)
+	}
+
+	nodeDossiers, err := checker.overlay.GetAll(ctx, nodeIDs)
```

Comment:
I think just `node` (`nodes`) is better var name than `nodeDossier` (`nodeDossiers`). 

It reads better in `node.Id`, `node.Operator.Wallet`, `node.Online()`, etc.